PULL_REQUEST_TEMPLATE.MD

```diff
@@ -20,7 +20,7 @@
 
 - [ ] I have verified that this change will not result in downtime. If it will, I have noted the impact below:
 
-- [ ] I have verified that no new dependencies were introduced. If they were, I have vetted them below:
+- [ ] I have verified that no new dependencies (excluding non production configurations such as test) were introduced. If they were, I have vetted them below:
```

Comment:
This is too specific to gradle probably... up for a better way of stating "only dependencies that matter"